### PR TITLE
Close settings with key back

### LINKED ISSUE #4678: Escape key to exit settings
I often try to exit the settings page using the escape key, as is convention in many other apps, such as discord and steam. I suggest the implementation of this feature, as I don't see any other use for the escape key in this situation.

### PR DESCRIPTION
**Pull request by cinqmilleans**
**Resolved / Related Issues**
Use key escape for close settings
Closes #4678

### CODE REVIEW

File: Files/Views/Settings.xaml.cs [hunk 1]
```diff
@@ -29,6 +30,8 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
             var currentView = SystemNavigationManager.GetForCurrentView();
             currentView.BackRequested += OnBackRequested;
 
+            Window.Current.CoreWindow.Dispatcher.AcceleratorKeyActivated += CoreDispatcher_AcceleratorKeyActivated;
```
Comment (d2dyno1) [inline Files/Views/Settings.xaml.cs:33]: Could that be done in MainPage.xaml?

Comment (cinqmilleans) [reply Files/Views/Settings.xaml.cs:33]: It’s technically possible. But why ?
This is a behavior of the Settings page, It should be in there.
In addition, we can manage other shortcuts later, to use the settings entirely from the keyboard.

**Comment by Jaiganeshkumaran** [comment]
Please don't use escape for back

**Comment by Jaiganeshkumaran** [comment]
VirtualKey.Back seems the backspace key which will cause issues with text boxes inside the page. Maybe use VirtualKey.GoBack?

**Comment by yaichenbaum** [comment]
> VirtualKey.Back seems the backspace key which will cause issues with text boxes inside the page. Maybe use VirtualKey.GoBack?

This shouldn't be an issue if it's properly handled.

**Pull request closed without merging**